package.json

```diff
@@ -45,6 +45,7 @@
     "@babel/preset-typescript": "^7.18.6",
     "@testing-library/jest-dom": "^5.16.4",
     "@types/jest": "^28.1.6",
+    "@types/multer": "^1.4.7",
```

Comment:
this shoudl probs be in the backend package for consistency but it doesnt really matter